### PR TITLE
Fix apk release issue for new Flutter Gallery

### PR DESCRIPTION
**Pull request by JoseAlba**
There was an issue when releasing an APK file onto the app store. The APK file would break on launch. This PR fixed that problem.

### CODE REVIEW

File: gallery/gallery/android/app/build.gradle [hunk 1]
```diff
@@ -38,7 +38,7 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
```
Comment (rami-a) [inline gallery/gallery/android/app/build.gradle]: Remove this TODO

**Comment by RedBrogdon** [comment]
Drive-by comment:

If it's still possible to change the package name, we generally use dev.flutter.* these days, since it better matches our domain (flutter.dev).

**Comment by rami-a** [comment]
Since this replaced the old gallery on the play store, I _think_ it needs to use the same package name. @mklim Do you know if this is the case?

**Comment by mklim** [comment]
Yeah, the package name needs to stay the same. https://stackoverflow.com/a/17582393/3968079

**Pull request merged**